examples/example-accordionpanel/rollup.config.js

```diff
@@ -0,0 +1,12 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
```

Comment:
Would you mind refactoring all rollup config (including the one currently set in the packages) to target common files at the root level as it is done for typescript and api-extractor configuration. This will ease maintenance.